src/player.js

```diff
@@ -1129,6 +1129,8 @@ export default class Player extends FakeEventTarget {
     this._readyPromise = new Promise((resolve, reject) => {
       this._eventManager.listen(this, CustomEvents.TRACKS_CHANGED, resolve);
       this._eventManager.listen(this, Html5Events.ERROR, reject);
+    }).catch(e => {
+      this.dispatchEvent(e);
```

Comment:
It looks like we are throwing the error event in multiple places, please verify that we only throw it once